src/main/java/seedu/fast/logic/commands/DeleteAppointmentCommand.java

```diff
@@ -0,0 +1,86 @@
+package seedu.fast.logic.commands;
+
+import static seedu.fast.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.List;
+
+import seedu.fast.commons.core.Messages;
+import seedu.fast.commons.core.index.Index;
+import seedu.fast.logic.commands.exceptions.CommandException;
+import seedu.fast.model.Model;
+import seedu.fast.model.person.Appointment;
+import seedu.fast.model.person.Person;
+
+public class DeleteAppointmentCommand extends Command {
+
+    public static final String COMMAND_WORD = "dappt";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Deletes appointment with the person identified"
+            + " by the index number used in the last person listing.\n\n"
+            + "Parameters (to add an appointment): \nINDEX (must be a positive integer), "
+            + "Example: \n" + COMMAND_WORD + " 1 ";
+
+    public static final String MESSAGE_DELETE_APPOINTMENT_SUCCESS = "Deleted appointment with %1$s";
+    public static final String MESSAGE_DELETE_APPOINTMENT_FAILED = "No appointment with %1$s yet!";
```

Comment:
It is not very clear what is the function of the string "%1$s", maybe you could create a constant that labels it, then append it to these strings? I noticed this in other classes as well.